### ISSUE TITLE
rubygems-bundler not allowing gems in 'global' to be executed, even if they don't exist in Gemfile

### ISSUE THREAD

**Issue by ghost** [rvm/rubygems-bundler#12]
@mpapis

I can't use gems when I'm in a project, that exist in global. they positively HAVE to be in the Gemfile, or I can't use them. Is that supposed to happen? (Blocking global gems if they're NOT in the Gemfile?) 

I thought it was supposed just cause limitation of use to the version of gems listed in Gemfile.lock, not block those not listed all together. https://gist.github.com/2338630 - I have github gem in 1.9.3-p125@global gemset. Its NOT listed in the project's Gemfile. I should be able to run that gem from the commandline without rubygems-bundler blocking its use since its NOT in the Gemfile.

e.g but it SHOULD be selecting the version of tzinfo i DO have in the Gemfile.lock (i have 2 versions of tzinfo installed, 
the older one is in the Gemfile.lock) and that it does do correctly.

So basically, in cucumber feature format:

``` feature
Given a gem is not in a project's Gemfile
And the gem is in the global gemset
Then rubygems-bundler should not attempt to execute it within the project's context
And rubygems-bundler should not block its command-line use
```
